cmds/mv/mv_test.go

```diff
@@ -43,15 +59,20 @@ func setup() (string, error) {
 	}
 
 	for _, t := range tests {
-		if err := ioutil.WriteFile(filepath.Join(d, t.n), []byte("hi"), t.m); err != nil {
+		if err := ioutil.WriteFile(filepath.Join(d, t.n), []byte(t.c), t.m); err != nil {
 			return "", err
 		}
 	}
 
 	return d, nil
 }
 
-func Test_mv_1(t *testing.T) {
+func resetFlags() {
+	// TODO: Is it possible to extract the default out of the declaration?
+	*update = false
```

Comment:
Don't do this. Rather, you have the test spawn new processes. As an example, look at the usage of testutil in: https://github.com/u-root/u-root/blob/master/cmds/wc/wc_test.go#L35

Reply:
Oh, I did it just like the tests for `cp` and `seq` and I thought that it wasn't elegant.
You're right, that is way better.